button-card.js

```diff
@@ -1,239 +1,241 @@
-var LitElement = LitElement || Object.getPrototypeOf(customElements.get("hui-error-entity-row"));
-var html = LitElement.prototype.html;
-
-class ButtonCard extends LitElement {
-  static get properties() {
-    return {
-      hass: Object,
-      config: Object,
-    };
-  }
+((LitElement, ButtonBase) => {
+  var html = LitElement.prototype.html;
+  var css = LitElement.prototype.css;
+
+  customElements.define(
+    'button-card-button',
+    class extends ButtonBase {
+      static get styles() {
+        return css`
+          ${ButtonBase.styles}
+          .mdc-button {
+            height: auto;
+            padding: 0;
+            color: inherit !important;
+          }
+        `
+      }
+    },
+  );
+
+  class ButtonCard extends LitElement {
+    static get properties() {
+      return {
+        hass: Object,
+        config: Object,
+      };
+    }
 
-  render() {
-    const state = this.__hass.states[this.config.entity];
-    switch (this.config.color_type) {
-      case 'blank-card':
-        return this.blankCardColoredHtml(state, this.config);
-      case 'label-card':
-        return this.labelCardColoredHtml(state, this.config);
-      case 'card':
-        return this.cardColoredHtml(state, this.config);
-      case 'icon':
-      default:
-        return this.iconColoredHtml(state, this.config);
+    static get styles() {
+      return css`
+        ha-icon {
+          display: flex;
+          margin: auto;
+        }
+        button-card-button {
+          display: flex;
+          margin: auto;
+          text-align: center;
+        }
+        button-card-button div {
+          padding: 8px;
```

Comment:
```suggestion
          padding: 4%;
```
I agree with @DavidFW1960. Let's change this to 4%

Reply:
Let's also add
```suggestion
          text-transform: none;
          font-size: 1.2rem;
```

Reply:
@peterhoe `text-transform: none;` will remove the capitalization